pkg/features/features.go

```diff
@@ -73,4 +79,5 @@ var defaultServiceCatalogFeatureGates = map[utilfeature.Feature]utilfeature.Feat
 	AsyncBindingOperations:  {Default: false, PreRelease: utilfeature.Alpha},
 	NamespacedServiceBroker: {Default: false, PreRelease: utilfeature.Alpha},
 	ResponseSchema:          {Default: false, PreRelease: utilfeature.Alpha},
+	DisableInstanceLocking:  {Default: false, PreRelease: utilfeature.Alpha},
```

Comment:
I would recommend changing this feature flag to be called `InstanceLocking` and default it to true.

The reason why is there are so many bugs in the history of feature flags that relate to people not understanding what it means to false a disableFoo flag. People are super bad at understanding the negative in the flag name. It is cleaner to just call it a flag for a feature and enable (true) flag means on, disable (false) means off. The feature is Instance Locking, and we default the feature on (true) (enabled).

Reply:
Good point @n3wscott, how about `OriginatingIdentityLocking`, since this will eventually apply to bindings too.